articles/server-apis/webapi-owin/02-authorization.md

```diff
@@ -35,7 +35,7 @@ Subsequently when the client passes along the `access_token` when making request
 
 ## 1. Define scopes
 
-You will need to define to list of scopes available in your application. In the [APIs section]("${manage_url}/#/apis) of the Auth0 Dashboard, select your API and then go to the __Scopes__ tab of the API. Configure the list of Scopes available in your API:
+You will need to define to list of scopes available in your application. In the [APIs section](https://manage.auth0.com/#/apis) of the Auth0 Dashboard, select your API and then go to the __Scopes__ tab of the API. Configure the list of Scopes available in your API:
```

Comment:
Please change this to `${manage_url}/#/apis`. This is the variable we use for the dashboard.